### ISSUE TITLE
improve error messages

### ISSUE THREAD

**Issue by Thomas-Ulrich** [SeisSol/SeisSol#43]
Currently when the free surface is not tagged as free surface and we try to run the code, we get the following error:

(...)
Thread 1 (Thread 0x2b5bb53a0de0 (LWP 20384)):
#0  0x00002b5bb0131eaf in waitpid () from /lib64/libc.so.6
#1  0x00002b5bb00c5cd1 in do_system () from /lib64/libc.so.6
#2  0x00002b5bb00c607c in system () from /lib64/libc.so.6
#3  0x00002b5bb584d366 in pm_linux_print_coredump () from /opt/ibmhpc/pe1404/base/intel/lib64/libpoe.so
#4  0x00002b5bb584c8ab in pm_lwcf_signal_handler () from /opt/ibmhpc/pe1404/base/intel/lib64/libpoe.so
#5  <signal handler called>
#6  0x00000000004ebbfc in seissol::initializers::time_stepping::LtsLayout::getCellInformation(CellLocalInformation*, unsigned int*&, unsigned int&) ()
#7  0x00000000006e66c2 in c_interoperability_initializeClusteredLts ()
#8  0x00000000004aa6ee in dg_setup_mod_mp_inigalerkin3d_us_level2_new_ ()
#9  0x00000000004a60ee in ini_seissol_mod_mp_ini_seissol_ ()
#10 0x0000000000700e00 in fortran_main ()
#11 0x0000000000747a06 in main ()

Can we make such error message more readable?

Thomas.

**Comment by ravil-mobile** [comment]
Hi Thomas, 
Could you, please, send me your particular setup to reproduce exactly this error? Today, I tried to reproduce the error; I took tpv5 test case and deleted physical surface #27. In this case,  I could see that the error was intercepted and an error message appeared but (let's say) too late, namely: during the LTS tree building process. But, obviously, we have to handle any errors like this during mesh reading.

**Comment by Thomas-Ulrich** [comment]
Hi,
Sorry for late answer.
So I ve tested TPV5 without free surface.
by commenting this line:
Physical Surface(101) = {1};

Then I create the mesh using this procedure:
```
gmsh -3 tpv5_f200m.geo -optimize -format msh2
gmsh2gambit -i  tpv5_f200m.msh -o tpv5_f200m.neu
pumgen tpv5_f200m.neu tpv5_f200m
```
and run TPV5 and get the following unspecfic error:

```
Mon Jun 03 16:17:23, Info:  maximum theoretical speedup (compared to GTS): 7.41631 per cell LTS, 5.02256 with the used clustering.
Mon Jun 03 16:17:23, Info:  Number of elements in dynamic rupture time clusters:
Mon Jun 03 16:17:23, Info:  0 (dr): 19
Mon Jun 03 16:17:23, Info:  1 (dr): 8940
Mon Jun 03 16:17:23, Info:  2 (dr): 85
Mon Jun 03 16:17:23, Info:  3 (dr): 0
Mon Jun 03 16:17:23, Info:  4 (dr): 0
Mon Jun 03 16:17:23, Info:  5 (dr): 0
Mon Jun 03 16:17:23, Info:  6 (dr): 0
srun: error: i01r05c04s11: task 5: Segmentation fault (core dumped)
srun: Terminating job step 133341.0
srun: error: i01r05c05s11: task 11: Segmentation fault (core dumped)
srun: error: i01r05c05s12: task 12: Segmentation fault (core dumped)
srun: error: i01r05c04s07: task 4: Segmentation fault (core dumped)
srun: error: i01r05c03s07: task 0: Segmentation fault (core dumped)
srun: error: i01r05c04s06: task 3: Segmentation fault (core dumped)
srun: error: i01r05c05s06: task 8: Segmentation fault (core dumped)
srun: error: i01r05c03s08: task 1: Segmentation fault (core dumped)
srun: error: i01r05c06s02: task 14: Segmentation fault (core dumped)
srun: error: i01r05c05s07: task 9: Segmentation fault (core dumped)
srun: error: i01r05c04s05: task 2: Segmentation fault (core dumped)
srun: error: i01r05c06s01: task 13: Segmentation fault (core dumped)
srun: error: i01r05c06s03: task 15: Terminated
srun: error: i01r05c05s02: task 6: Terminated
srun: error: i01r05c05s03: task 7: Terminated
srun: error: i01r05c05s09: task 10: Terminated
srun: Force Terminated job step 133341.0
```
Note that (surprisingly) if I use BC_fs = 0 in namelist Boundaries I get a similar error.

Note that I also tried to remove the absorbing boundary condition instead of the free surface, and got the same results.

Thomas.

**Comment by Thomas-Ulrich** [comment]
closed as duplicate of #160
